engine/src/cmd/unit_csv.cpp

```diff
@@ -1127,14 +1108,11 @@ void Unit::LoadRow(std::string unit_identifier, string modification, bool saved_
 
 CSVRow GetUnitRow(string filename, bool subu, int faction, bool readlast, bool &rread) {
```

Comment:
This function is not called. You can delete it rather than refactor. It's a good practice in this project to check if the function is used. As I refactor, code gets orphaned (?).  Also, the original devs left quite a few instances of stuff in progress that wasn't actually complete. Obviously, it would be much harder for us to know where they were going with it today.

Reply:
OK, I can delete it.

Reply:
Deleted.